Uchu.World/Handlers/WorldInitializationHandler.cs

```diff
@@ -180,8 +180,11 @@ public async Task PlayerLoadedHandler(PlayerLoadedMessage message, Player player
         /// <param name="session">The session cache for the connection</param>
         private async Task SendCharacterXmlDataToClient(Character character, IRakConnection connection, Session session)
         {
+            await using var ctx = new UchuContext();
+            var gmLevel = ctx.Users.Where(i => i.Id == character.UserId).First().GameMasterLevel;
+            
             // Get the XML data for this character for the initial character packet
-            var xmlData = GenerateCharacterXmlData(character);
+            var xmlData = GenerateCharacterXmlData(character, gmlevel);
```

Comment:
This will crash now bc the variable name is incorrect.

Reply:
oh for